api/argo-templates/render_test.go

```diff
@@ -35,7 +35,7 @@ import (
 func TestRenderWorkerRebuildTemplate(t *testing.T) {
 	t.Run("It should render a workflow template for a group of worker nodes", func(t *testing.T) {
 		targetNcns := []string{"ncn-w006", "ncn-w005"}
-		_, err := GetWorkerRebuildWorkflow(targetNcns)
+		_, err := GetWorkerRebuildWorkflow(targetNcns, true)
```

Comment:
I'd maybe make some constants for dry run booleans to pass in for tests/calls so that true is doDryRun. Saves looking up what true is indicating in the calls.